test/integration/document/Thumbnails.e2e.test.js

```diff
@@ -21,7 +33,13 @@ describe('Preview Document Thumbnails', () => {
 
         cy.wait(301); // Wait for toggle animation to complete
 
-        return cy.getByTestId('thumbnails-sidebar');
+        const thumbnailSubject = cy.getByTestId('thumbnails-sidebar');
+
+        return {
```

Comment:
I wanted to expose methods to assert the visibility instead of having to copy paste the absurd `have.css...` assertion. Open to better ideas

Reply:
The current solution looks okay, although spreading the thumbnailsSubject into the return looks a bit odd. If you're looking for an alternative, the `.should` command can accept a function as an argument. You could do something like this:

```
const beVisible = ($thumbnailsSidebar) => {
    expect($thumbnailsSidebar).to.have.css(...etc.);
}

cy.getByTestId('thumbnails-sidebar').should(beVisible);
```

Reply:
I like that, thanks for the suggestion! I'll try it out